NeoMathEngine/src/MathEngineDnnDropout.h

```diff
@@ -20,9 +20,9 @@ limitations under the License.
 
 namespace NeoML {
 
-// Dropout descriptor
-struct CMathEngineDropoutDesc : public CDropoutDesc {
-	explicit CMathEngineDropoutDesc( IMathEngine& mathEngine, float rate, bool isSpatial, bool isBatchwise,
+// Dropout descriptor containing whole mask
+struct CMaskDropoutDesc : public CDropoutDesc {
```

Comment:
It should be checked, If class `CMaskDropoutDesc` is not called from anywhere now, it should be removed